tensorrt_llm/_torch/attention_backend/trtllm.py

```diff
@@ -1190,7 +1190,7 @@ def update_spec_dec_param(
                 dtype=torch.int,
                 device='cuda',
             )
-            if get_sm_version() >= 100:
+            if get_sm_version() >= 100 and get_sm_version() != 120:
```

Comment:
could we include 121 as well? 

```suggestion
            if get_sm_version() >= 100 and get_sm_version() not in [120, 121]:
```